createBenchmark.sh

```diff
@@ -36,5 +49,9 @@ cd "$benchmark"
 # calls OS specific sed inplace function
 sed_inplace 's/camunda\/zeebe/gcr.io\/zeebe-io\/zeebe/' zeebe-values.yaml
 sed_inplace "s/SNAPSHOT/$benchmark/" zeebe-values.yaml
+sed_inplace "s/starter:zeebe/starter:$benchmark/" starter.yaml
+sed_inplace "s/starter:zeebe/starter:$benchmark/" simpleStarter.yaml
+sed_inplace "s/starter:zeebe/starter:$benchmark/" timer.yaml
```

Comment:
just a thought: I we could also do it in one line but we can leave it as it is no real preference